### PR TITLE
De-duplicate demo app code

### PR DESCRIPTION
**Pull request by ecraig12345**
#### Pull request checklist

- [x] Include a change request file using `$ npm run change`

#### Description of changes

Add a shared `createDemoApp` utility and use it to bootstrap all the packages' demo apps.

Also use each package's readme as the content for the getting started page, rather than some duplicated content which didn't even apply to packages besides OUFR itself.

#### Focus areas to test

Verify each demo app looks/works okay (or no worse than it was before)

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/9114)

### CODE REVIEW

File: packages/experiments/src/demo/GettingStartedPage.tsx [hunk 1]
```diff
@@ -1,103 +1,17 @@
...
+const rootClass = mergeStyles({
+  marginTop: -20,
+  padding: 40
+});
+
+export const GettingStartedPage: React.StatelessComponent = () => {
+  return (
+    <div className={rootClass}>
+      {/* tslint:disable-next-line:deprecation */}
+      <PageMarkdown>{require<string>('!raw-loader!../../README.md')}</PageMarkdown>;
```
Comment (jordandrako) [inline packages/experiments/src/demo/GettingStartedPage.tsx]: A few of the Getting Started pages have an errant `;` in the content. Do a quick sweep for these and you should be good.
```suggestion
      <PageMarkdown>{require<string>('!raw-loader!../../README.md')}</PageMarkdown>
```

Comment (ecraig12345) [reply packages/experiments/src/demo/GettingStartedPage.tsx]: Good catch, thanks!

**Comment by jordandrako** [comment]
Quick non-blocking question. Why are you specifically using `PageMarkdown` instead of the new `Markdown` component?

**Comment by ecraig12345** [comment]
Mainly because it has the older non-Fluent theme, which is also what other parts of the demo app use.

**Pull request merged**